ironfish/src/telemetry/telemetry.ts

```diff
@@ -76,7 +77,9 @@ export class Telemetry {
     void this.flushLoop()
 
     if (this.metrics) {
-      void this.metricsLoop()
+      this.metricsInterval = setTimeout(() => {
```

Comment:
Adding this change so that we don't log metrics on initial node startup. This would cause (and was already causing) nodes to send messages with network traffic = 0 because there hasn't been any messages yet. This seems like it would lead to inaccurate data. We want to allow some collection time before messages are sent

Reply:
This is mostly just personal thought, but usually if you need a comment like this, it would be better to encode that explanation into logic. Maybe just check that we don't submit those metrics until a certain amount of time has passed and were connected to the network or something?